WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginNoSitesErrorViewModel.kt

```diff
@@ -0,0 +1,23 @@
+package org.wordpress.android.ui.accounts.login
+
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.withContext
+import org.wordpress.android.WordPress
+import org.wordpress.android.modules.BG_THREAD
+import org.wordpress.android.modules.UI_THREAD
+import org.wordpress.android.viewmodel.ScopedViewModel
+import javax.inject.Inject
+import javax.inject.Named
+
+class LoginNoSitesErrorViewModel @Inject constructor(
+    @Named(UI_THREAD) mainDispatcher: CoroutineDispatcher,
+    @Named(BG_THREAD) val bgDispatcher: CoroutineDispatcher
+) : ScopedViewModel(mainDispatcher) {
+    fun signOutWordPress(application: WordPress) {
+        launch {
+            withContext(bgDispatcher) {
+                application.wordPressComSignOut()
```

Comment:
@ashiagr _ lol, I didn't because that functionality is already tested in the MeViewModelTest ... our test would test the same exact functionality, but I'll add it.